_data/iamspathan/iamspathan.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sohail Pathan # No longer than 18 characters
+institution: Lovely Professional University # no longer than 58 characters
+profile_pic: sohail.png # Relative path to the image you uploaded
```

Comment:
image file path name seems incorrect to me, please change `sohail.png` to `sohail.jpg`

Reply:
changes made in last commit